src/game/BattleGround/BattleGroundWS.cpp

```diff
@@ -501,10 +424,8 @@ void BattleGroundWS::Reset()
 void BattleGroundWS::EndBattleGround(Team winner)
 {
     // win reward
-    if (winner == ALLIANCE)
-        RewardHonorToTeam(GetBonusHonorFromKill(m_HonorWinKills), ALLIANCE);
-    if (winner == HORDE)
-        RewardHonorToTeam(GetBonusHonorFromKill(m_HonorWinKills), HORDE);
+    RewardHonorToTeam(GetBonusHonorFromKill(m_HonorWinKills), winner);
```

Comment:
What if there is no winner? (45m time limit)

Reply:
Good point, thanks.